docs/contributing/tests.md

```
@@ -1,98 +1,135 @@
-+        uploader: int
-+        visibility: Visibility
-+    mock.return_value = Dataset(uploader=1, visibility=Visibility.PRIVATE)
-
-    _get_dataset_raise_otherwise(
-        dataset_id=1,
-        user=admin,
--        expdb=expdb_test,
-+        expdb=None,
+Include one test against the `py_api` that confirms a successful request has the expected result:
+
+```python
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)